boards/beagle/pocketbeagle_2/doc/index.rst

```diff
@@ -95,6 +145,9 @@ The binary will run and print Hello world to the MCU_UART0 port.
 Debugging
 *********
 
```

Comment:
A53 can be debugged via JTAG.

Reply:
I could not get it to work myself, so I have not added it in this PR.